tmail-backend/combined-identity/src/main/java/com/linagora/tmail/combined/identity/UsersRepositoryModuleChooser.java

```diff
@@ -45,7 +51,14 @@ public class UsersRepositoryModuleChooser {
     public static List<Module> chooseModules(Implementation implementation) {
         return switch (implementation) {
             case LDAP -> ImmutableList.of(new LdapUsersRepositoryModule());
-            case COMBINED -> ImmutableList.of(new CombinedUsersRepositoryModule());
+            case COMBINED -> ImmutableList.of(new CombinedUsersRepositoryModule(), new AbstractModule() {
+                @Provides
+                @Singleton
+                @Named(CombinedUserDAO.BACKPORT_INJECT_NAME)
+                public UsersDAO provideBackportUserDAO(CassandraUsersDAO cassandraUsersDAO) {
+                    return cassandraUsersDAO;
+                }
```

Comment:
Maybe extract this to a Module for better readability?

Reply:
Maybe the clean way here would have yes to separate code in multiple modules. combined-identity-api for generic code, then a module for cassandra implem, one for postgres implem?

It's ok to extract that work into an other ticket/PR as well